Storage/SQL/BrowserTable.swift

```diff
@@ -906,25 +881,25 @@ open class BrowserTable: Table {
         if from < 22 && to >= 22 {
             if !self.run(db, queries: [
                 "DROP TABLE IF EXISTS \(TablePageMetadata)",
-                attachedPageMetadataCreate,
-                attachedIndexPageMetadataCacheKeyCreate,
-                attachedIndexPageMetadataSiteURLCreate]) {
+                pageMetadataCreate,
```

Comment:
Note that no clients will ever have a `TablePageMetadata`, because it was created as `AttachedTablePageMetadata`, so there's no need to make one only to drop it.